### PR TITLE
Moved buy section into side bar and fixed the relevant css

### LINKED ISSUE #962: Move $buy section into grey side-box
Currently, on the `editions` page, the `$ Buy` section is outside / below the grey side-box which contains the main call to actions for the book. The proposal is to (instead) move these `buy` options into the side-box (see attached figures for reference)

# Before

screen shot 2018-05-18 at 18 15 23pmpdt

# After

Note: Instead of "Buy Now" perhaps we want to say "Buy this Book" (I'm open to whatever 'copy' is most effective at communicating the intent)

screen shot 2018-05-18 at 18 19 18pmpdt

@Num170R, @tabshaikh, @whatisgalen any of you interested in trying this one?

### PR DESCRIPTION
**Pull request by Num170R**
Closes #962 

## Description:
In this Pull Request we have made the following changes:
  * Modified some html code to move buy section into sidebar
  * Changed the relevant css code in order to be consistent

### CODE REVIEW

File: static/css/master.css [hunk 1]
```diff
@@ -1973,11 +1973,11 @@ div.Tools table td.vendor {padding:3px 10px 0 42px;}
 div.Tools table td.price {padding:3px 0 0 0;text-align:right;}
 div.Tools table.buy-options-table {
     border-spacing: 5px;
-    margin-left: 45px;
-    width: 200px;
+    margin-left: 0px;
+    width: 150px;
```
Comment (mekarpeles) [inline static/css/master.css]: I think we want to increase this to 175px. On the production site, we have an API key (which the dev clients may not have) which also fetches the price. When this happens, 150px is too small to contain the affiliate name and the price. e.g. https://dev.openlibrary.org/books/OL7389275M/Great_Physicists

**Comment by mekarpeles** [comment]
Looks great overall! 1 blocker (adjusting width) mentioned in code review

File: static/css/master.css (hunk 1 above)
Comment (Num170R) [reply static/css/master.css]: I had checked this manually adjusting the value. Give me some time to upload the relevant pic.

Comment (mekarpeles) [reply static/css/master.css]: It may also be slightly different on the /works versus the /books page(s)

**Comment by mekarpeles** [comment]
Looking good -- can you also apply this patch?

```
index bfd7152..ec206e3 100644
--- a/static/css/master.css
+++ b/static/css/master.css
@@ -5649,7 +5649,7 @@ div.tools-override {
 }

 div.tools-override .Tools .btn-notice{
-    width: 160px;
+    width: 180px;
     margin-left: 45px;
     font-size: .9em;
 }
```

You're right that `works` and `books` (i.e. editions) pages have different size sidebars. This patch will fix that so all of your changes look good in both views.

Then will happily merge. Thanks for this contribution!

**Comment by mekarpeles** [comment]
LGTM! Thank you

**Pull request merged**